shader/binning.wgsl

```diff
@@ -63,7 +64,18 @@ fn main(
     for (var i = 0u; i < N_SLICE; i += 1u) {
         atomicStore(&sh_bitmaps[i][local_id.x], 0u);
```

Comment:
Ah, I didn't realize that was a strong guarantee. In WebGPU world, it's probably worth skipping this explicit zeroing, but in native world it might be worth compiling with zeroing by infrastructure disabled, in which case we would need this.

Reply:
Good point - for us it's not impactful, but e.g. before https://github.com/linebender/vello/pull/363 this would have mattered for the MSL conversion